cloud/awsprovider.go

```diff
@@ -445,9 +511,9 @@ func (*AWS) GetDisks() ([]byte, error) {
 		return nil, err
 	}
 	defer clusterConfig.Close()
-	bytes, _ := ioutil.ReadAll(clusterConfig)
+	b, _ := ioutil.ReadAll(clusterConfig)
 	var clusterConf map[string]string
-	json.Unmarshal([]byte(bytes), &clusterConf)
+	json.Unmarshal([]byte(b), &clusterConf)
 	region := aws.String(clusterConf["region"])
 	c := &aws.Config{
 		Region:      region,
```

Comment:
It's hiding just below this diff marker, but similar comment about omitting `Credentials` in order to have the system correctly iterate through all known credential providers

Reply:
Done